Gemfile

```diff
@@ -14,6 +14,7 @@ gem 'flipper-ui',        '~> 0.23' # UI for the Flipper gem
 gem 'jbuilder',          '~> 2.11' # Build JSON APIs with ease [https://github.com/rails/jbuilder]
 gem 'pg',                '~> 1.1'  # Use postgresql as the database for Active Record
 gem 'puma',              '~> 5.0'  # Use the Puma web server [https://github.com/puma/puma]
+gem 'pundit'
```

Comment:
I believe we're using the pattern of pessimistically locking the gems in the Gemfile to the latest minor version. Also if you could, add a small comment summarizing what the gem does